### DISCUSSION TITLE
[Feature Request] File Conversion

### DISCUSSION THREAD

**Discussion by WillMontgomery** [diced/zipline#353] [Ideas]
I would like to see a page that could be used to convert files. For example, a URL such as https://example.com/convert which would display a page to upload, convert, and download files or save them to the Zipline files library. This would be a very convenient solution as other free sites out there have limitations for how many files that can be converted per day, etc. Additionally, this capability would vastly expand the function of Zipline. While it would likely require a lot of resources to implement, there aren't any [good] self-hosted alternatives that I could find with this function, so it would definitely stand out as a nice-to-have.

**Comment by TacticalTechJay** [comment]
What sort of things would you often see yourself converting?

**Comment by xSkeletor** [comment]
This would be a cool feature, but it may be out of scope for this project since it doesn't have anything to do with uploading files themselves. But with that being said, I still would also like to see this become a feature for the same reasons that you explained.

**Comment by WillMontgomery** [comment]
Personally, I would be converting .jpg and .png files to .webp for my use
case primarily, but it may also be neat to convert multimedia files as
well. Really just depends how far someone would develop it.

**Reply by ptlthg** [reply]
I agree, being able to convert between just these three would be enough for me

**Comment by mrslugmoon** [comment]
This would be amazing, as im using free online converters everyday.
